src/core_plugins/kibana/public/management/sections/objects/components/objects_table/objects_table.js

```diff
@@ -0,0 +1,315 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { flattenDeep } from 'lodash';
+import { Header } from './components/header';
+import { Flyout } from './components/flyout';
+import { Relationships } from './components/relationships';
+import { Table } from './components/table';
+
+import { EuiSpacer, EuiHorizontalRule, Query } from '@elastic/eui';
+import { retrieveAndExportDocs } from '../../lib/retrieve_and_export_docs';
+import { scanAllTypes } from '../../lib/scan_all_types';
+import { saveToFile } from '../../lib/save_to_file';
+import { getQueryText } from '../../lib/get_query_text';
+import { getSavedObjectIcon } from '../../lib/get_saved_object_icon';
+import { ensureMinimumTime } from '../../../indices/create_index_pattern_wizard/lib/ensure_minimum_time';
+import { getRelationships } from '../../lib/get_relationships';
+
+export const INCLUDED_TYPES = ['index-pattern', 'visualization', 'dashboard', 'search'];
```

Comment:
Is there a way to dynamically generate this list with a terms aggregation on `type`? That way all registered saved object types will be displayed, like timelion sheets

Reply:
We could possibly, but we'd need to expand the relationships code to support more types since those are all handled individually. We could probably show all saved objects, but only show relationships for a known whitelist. Thoughts?

Reply:
So I looked into this a bit and we can't even import those through the normal flow, which uses [a registry](https://github.com/elastic/kibana/search?utf8=%E2%9C%93&q=savedObjectManagementRegistry&type=) to indicate which types have import flows. This PR adds support for index patterns since we really wanted that support here so we made the relevant forks in the code to handle that type differently than other saved objects and I'm not sure we want to spend the time doing that for other types at this time.

@jen-huang Thoughts?

Reply:
Timelion sheets looks to be registered through `savedObjectManagementRegistry`, so it should be import/export-able, right?

I think we should show all object types that are registered (+ index pattern), but only show relationships for a subset whitelisted types, since finding relationships for a saved object is custom per type.

It would be great if `savedObjectManagementRegistry` had a way to define a relationship structure upon calling `.register()`, but I think that's beyond the scope of this PR.

Reply:
Hmm, you're right. This seems more complicated because it's in a separate plugin so the registry doesn't know about it unless the file has been loaded which it isn't on the management page. I'll make a follow up ticket to look more into this